src/views/Dashboard/Create.vue

```diff
@@ -97,7 +97,7 @@ export default {
       delete localStorage.content
       console.log(localStorage.content)
     } else {
-      fetch("http://localhost:3000/:id/View/:id")
+      fetch("https://speedwagonmailback.herokuapp.com/:id/View/:id")
```

Comment:
linknya samain sama si elvin bikin we di 
https://github.com/A-WebProgramming-Kelompok-9/EmailBackend/blob/master/index.js

Reply:
buat create email, si elvin bikin ini 

app.post("/email/add", (req, res) => {
    emaildb.insertEmail(req.body.username,req.body.receiver,req.body.title
        ,req.body.content,req.body.attachment).then((ress, err) => {
        res.send(ress)
    });
})

samain parameter sama linknya we